src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './style/Calculator.css';
+
+const Calculator = () => (
+  <div id="calculator-body">
+    <div id="calculator-screen">
+      <p>0</p>
+    </div>
+    <div id="calculator-buttons">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="orange-button">
+        ÷
+      </button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="orange-button">
+        x
+      </button>
+      <button type="button">4</button>
+      <button type="button">5</button>
```

Comment:
According to the requirement, there should be a child component within this Calculator component.

for example:

```
Function Calculator() {
  return (
    <SomeComponents />
    // YOUR JSX GOES HERE
  )
}
```

Consider doing this in other to stick to the project requirements.